content/en/docs/tasks/administer-cluster/change-pv-access-mode-readwriteoncepod.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Change the Access Mode of a PersistentVolume to ReadWriteOncePod
+content_type: task
+weight: 90
+min-kubernetes-server-version: 1.22
+---
+
+<!-- overview -->
+
+This page shows how to change the access mode on an existing PersistentVolume to
+use `ReadWriteOncePod`.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{< note >}}
+The `ReadWriteOncePod` access mode was introduced as an alpha feature in
+Kubernetes 1.22. To use this feature in Kubernetes 1.22, you need to enable the
+`ReadWriteOncePod` feature gate manually.
+{{< /note >}}
```

Comment:
```suggestion
```
I'd skip that part, provided that the feature is on by default in Kubernetes v1.25
Why that version? Once this change ships, v1.25 will be the oldest version with available docs.

Reply:
The feature was still in alpha as of 1.25 and so this may still be applicable. What wording would you use for this?

Reply:
```suggestion
{{< note >}}
The `ReadWriteOncePod` access mode graduated to stable in the Kubernetes v1.29 release.
If you are running a version of Kubernetes older than v1.29, you might need to enable a
feature gate. Check the documentation for your version of Kubernetes.
{{< /note >}}
```